README.md

```diff
@@ -80,7 +80,7 @@
 | [route](https://www.ldoceonline.com/dictionary/route) [🔊](https://www.ldoceonline.com/media/english/ameProns/route1.mp3) | ✅ [ruːt] |  |
 | [rust](https://www.ldoceonline.com/dictionary/rust)[🔊](https://www.ldoceonline.com/media/english/ameProns/rust1.mp3) | ✅ [rʌst] | ❌ [rust] |
 | [safari](https://www.ldoceonline.com/dictionary/safari) [🔊](https://www.ldoceonline.com/media/english/ameProns/safari1.mp3) | ✅ [sə'fɑːrɪ] | ❌ [sæfərɪ] |
-| [scheme](https://www.ldoceonline.com/dictionary/scheme) [🔊](https://www.ldoceonline.com/media/english/ameProns/scheme1.mp3) | ✅ [skiːm] | ❌ [s'kæmə] |
+| [schema](https://www.ldoceonline.com/dictionary/schema) [🔊](https://www.youtube.com/watch?v=9zvXH8Rzk7I) | ✅ [s'kæmə]  | ❌ [skiːm] |
```

Comment:
Should it pronounce as ``[`skimə]`` ?

Reply:
Yes, schema. 
Scheme is wrong one.

Reply:
I mean the schema should be pronounced as ``[`skimə]``